### PR TITLE
docs: Add bi-weekly report (0828-0910)

### PR DESCRIPTION
**Pull request by saltism**
Changes:

Add bi-weekly report (0828-0910).

### CODE REVIEW

File: blog/en/blog/2023/09/13/biweekly-report.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+> We have recently made fixes and improvements to certain features of Apache APISIX and APISIX Ingress Controller. These include providing native JSON data structure input support for the `wasm` plugin, adding UNIX socket listening support for the Changtin Tech `WAF` plugin, and introducing the ability to remove the etcd dependency required by APISIX within the APISIX Ingress to reduce architectural complexity. For more details, please refer to the biweekly report.
+
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+From 8.28 to 9.10, 15 contributors submitted 32 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
```
Comment (yzeng25) [inline blog/en/blog/2023/09/13/biweekly-report.md]: zh-cn `16 位开发者为 Apache APISIX 提交了 47 个 commit`

Comment (yzeng25) [reply blog/en/blog/2023/09/13/biweekly-report.md]: I believe 16 and 47 is correct, please update here.

File: blog/zh/blog/2023/09/13/biweekly-report.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/09/11/fQoyADFJ_contributors%20list%201-0911.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/09/11/UVL4ZB8S_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5-0911.png)
+
+## 近期亮点功能
+
+### Apache APISIX
+
+- [为 `wasm` 插件提供原生 json 数据结构入参支持](https://github.com/apache/apisix/pull/10072)（贡献者：[Sn0rt](https://github.com/Sn0rt))
```
Comment (yzeng25) [inline blog/zh/blog/2023/09/13/biweekly-report.md]:
```suggestion
- [为 `wasm` 插件提供原生 json 数据结构入参支持](https://github.com/apache/apisix/pull/10072)（贡献者：[Sn0rt](https://github.com/Sn0rt)）
```

File: blog/en/blog/2023/09/13/biweekly-report.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## Recent Blog Recommendations
+
+- [Release Apache APISIX 3.5.0](https://apisix.apache.org/blog/2023/09/01/release-apache-apisix-3.5.0/)
+
+  We are pleased to present Apache APISIX 3.5.0 with exciting new features and improvements to user experiences.
+
+- [Coraza: Elevating APISIX with Cutting-Edge WAF Features](https://apisix.apache.org/blog/2023/09/08/APISIX-integrates-with-Coraza/)
+  
+  The integration of APISIX and Coraza provides reliable security protection and ensures the integrity and reliability of API services.
```
Comment (yzeng25) [inline blog/en/blog/2023/09/13/biweekly-report.md]:
```suggestion
- [Coraza: Elevating APISIX with Cutting-Edge WAF Features](https://apisix.apache.org/blog/2023/09/08/APISIX-integrates-with-Coraza/)
  
  The integration of APISIX and Coraza provides reliable security protection and ensures the integrity and reliability of API services.
  
- [Release Apache APISIX 3.5.0](https://apisix.apache.org/blog/2023/09/01/release-apache-apisix-3.5.0/)

  We are pleased to present Apache APISIX 3.5.0 with exciting new features and improvements to user experiences.
```

Comment (yzeng25) [reply blog/en/blog/2023/09/13/biweekly-report.md]: align w/ zh-cn doc, put coraza blog up front

File: blog/zh/blog/2023/09/13/biweekly-report.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+3. 支持在 APISIX Ingress 中移除 APISIX 自身需要的 etcd，减少架构复杂度（实验特性）
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/09/11/fQoyADFJ_contributors%20list%201-0911.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/09/11/UVL4ZB8S_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5-0911.png)
```
Comment (Yilialinn) [inline blog/zh/blog/2023/09/13/biweekly-report.md]: wrong picture and bad link
New contributors: SuperBo,zhangyuthink,JYan00,zclaiqcc,utkar-sh-ukla

File: blog/zh/blog/2023/09/13/biweekly-report.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+2. 为长亭 `WAF` 插件增加 UNIX sock 监听支持
+
+3. 支持在 APISIX Ingress 中移除 APISIX 自身需要的 etcd，减少架构复杂度（实验特性）
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/09/11/fQoyADFJ_contributors%20list%201-0911.png)
```
Comment (Yilialinn) [inline blog/zh/blog/2023/09/13/biweekly-report.md]: The names are too close and not at the same height. Please modify it a little bit.

File: blog/en/blog/2023/09/13/biweekly-report.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+From 8.28 to 9.10, 15 contributors submitted 32 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+
+We have recently fixed and improved some features, and the summary of the updates is as follows:
+
+1. Provide native JSON data structure input support for the `wasm` plugin
+
+2. Add UNIX socket listening support for the Changtin Tech `WAF` plugin
```
Comment (Yilialinn) [inline blog/en/blog/2023/09/13/biweekly-report.md]:
```suggestion
2. Add UNIX socket listening support for the `chaitin-waf` plugin
```

Comment (Yilialinn) [reply blog/en/blog/2023/09/13/biweekly-report.md]: check the whole passage and revise accordingly

File: blog/zh/blog/2023/09/13/biweekly-report.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/09/11/Co3naagN_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E4%B8%AD%E6%96%87-0911.png
+---
+
+> 我们近期对 Apache APISIX 和 APISIX Ingress Controller 的部分功能进行了修复或改善，包括为 `wasm` 插件提供原生 json 数据结构入参支持、为长亭 `WAF` 插件增加 UNIX sock 监听支持、以及支持在 APISIX Ingress 中移除 APISIX 自身需要的 etcd，减少架构复杂度。详情请阅读本期双周报。
```
Comment (Yilialinn) [inline blog/zh/blog/2023/09/13/biweekly-report.md]:
```suggestion
> 我们近期对 Apache APISIX 和 APISIX Ingress Controller 的部分功能进行了修复或改善，包括为 wasm 插件提供原生 json 数据结构入参支持，为长亭 WAF 插件增加 UNIX sock 监听支持，以及支持在 APISIX Ingress Controller 中移除 APISIX 自身需要的 etcd，减少架构复杂度。详情请阅读本期双周报。
```

File: blog/zh/blog/2023/09/13/biweekly-report.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+从 2023.8.28 - 2023.9.10，有 16 位开发者为 Apache APISIX 提交了 47 个 commit。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+
+我们近期对部分功能进行了修复或改善，更新内容汇总如下：
+
+1. 为 `wasm` 插件提供原生 json 数据结构入参支持
+
+2. 为长亭 `WAF` 插件增加 UNIX sock 监听支持
```
Comment (Yilialinn) [inline blog/zh/blog/2023/09/13/biweekly-report.md]:
```suggestion
1. 为 wasm 插件提供原生 json 数据结构入参支持

2. 为 `chatin-waf` 插件增加 UNIX sock 监听支持
```

**Review by yzeng25 — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Comment by yzeng25** [comment]
@JYan00 @Yilialinn could you update the attached image in another pull request? The text 'Thank your for contributing to' is missing.

image

**Comment by saltism** [comment]
> @JYan00 @Yilialinn could you update the attached image in another pull request? The text 'Thank your for contributing to' is missing.
> 
> image

Sorry, already fixed that.

**Comment by yzeng25** [comment]
> > @JYan00 @Yilialinn could you update the attached image in another pull request? The text 'Thank your for contributing to' is missing.
> > image
> 
> Sorry, already fixed that.

Great, I see modifications in #1695 and #1696.

**Comment by yzeng25** [comment]
> > > @JYan00 @Yilialinn could you update the attached image in another pull request? The text 'Thank your for contributing to' is missing.
> > > image
> > 
> > 
> > Sorry, already fixed that.
> 
> Great, I see modifications in #1695 and #1696.

@JYan00 what is your plan on fixing this? #1695 and #1696 are closed just now.

**Pull request merged**